challenges/identifying-shared-costs.md

```diff
@@ -52,11 +52,15 @@ If you have a strong handle on these subjects, continue on to better understand
 
 <span id="why-allocate"></span>
 
+## Inntroduction: what are shared costs?
+
+Examining two completely opposite (and over simplified) examples of cloud costs will illustrate what are and are not shared costs. Example #1: A web application is run on a single compute instance with both the web server and the datbase running locally. The compute instance only hosts this one, simple web application. As a general practice, the enterprise running the web application charges back costs based on product. In this case, the costs for both the compute instance and its attached block storage is not a shared cost: they are both charged back to the product owner of the single Web applicaiton running on the compute instance. Example #2: a single compute instance hosts a number of compute containers. The containers running on the compute instance run components of several different web applications and serve several different customers. The cloud provider bills the enterprise for the compute instance's run time and for the allocated block storage attached to the compute instance. The enterprise running these web applications charges back costs based on product. Because several different products run on the single computee instance, the costs for its run time and attached storage are shared costs for the enterprisee, which would like to develop an approach for splitting up the cost for the compute instance and charging the split costs back to the product owners.
```

Comment:
I like the idea of "What are shared costs" before the "why allocated shared costs", but I have some thoughts.

Do you think this gets too technical too fast?  You go fairly technical with what a shared cost is, and then we go back to more high level in the following paragraph.

I think I'd also like the examples to be separated out into separate lines, but perhaps we can be even more high level than using something along the lines of saying a "compute instance". That's why I use the pizza example below to show what can be considered a shared cost... 

Are the two examples redundant?  Should we move the pizza example up to the beginning as "the venue is an example of a shared cost"?  I think that might flow better, because I think the second paragraph also has a lot of information -- it goes deep (with good data!) and then pulls back again.  Thoughts?